src/styles/prose.scss

```diff
@@ -43,3 +43,22 @@
 .prose td {
   @apply dark:text-dark-text;
 }
+.prose pre button.copy-icon div.copy-icon {
+  opacity: 1;
+  transition: opacity 0.3ms ease-in-out;
+}
+.prose pre button.copy-icon div.check-icon {
+  opacity: 0;
+  transition: opacity 0.3ms ease-in-out;
+}
+.prose pre button.check-icon div.check-icon {
+  opacity: 1;
+  transition: opacity 0.3ms ease-in-out;
+}
+.prose pre button.check-icon div.copy-icon {
+  opacity: 0;
+  transition: opacity 0.3ms ease-in-out;
+}
+.prose pre button {
+  overflow: hidden;
+}
```

Comment:
i can't see any animations on clicks

Reply:
move all this styles to tailwind classes to app.js to part where you add and handle copy button clicks